specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -13221,6 +16065,111 @@
         }
       }
     },
+    "FunctionEnvelope": {
+      "description": "Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "FunctionEnvelope resource specific properties",
+          "properties": {
+            "name": {
+              "description": "Function name.",
+              "type": "string"
+            },
+            "functionAppId": {
```

Comment:
Function name isn't read-only in the [Kudu sources](https://github.com/projectkudu/kudu/blob/master/Kudu.Contracts/Jobs/JobBase.cs).